node/util.go

```diff
@@ -57,7 +58,19 @@ func PVToJSON(obj tmtypes.PrivValidator) ([]byte, error) {
 }
 
 func PVToFile(path string, perm os.FileMode, obj tmtypes.PrivValidator) error {
-	data, err := PVToJSON(obj)
+	return writeConfigIfNotExist(path, perm, obj)
+}
+
+func NodeKeyToJSON(obj *p2p.NodeKey) ([]byte, error) {
+	return cdc.MarshalJSON(obj)
+}
+
+func NodeKeyToFile(path string, perm os.FileMode, obj *p2p.NodeKey) error {
```

Comment:
func name will be used as node.NodeKeyToFile by other packages, and that stutters; consider calling this KeyToFile